### DISCUSSION TITLE
How can I disable the dimmed text preview of completions?

### DISCUSSION THREAD

**Discussion by ghost** [hrsh7th/nvim-cmp#1946] [Q&A] [answered]
It's quite annoying when I'm just trying to type something out and then the preview for completions pushes what I am writing ahead. How can I just disable the dimmed text preview for completions without entirely removing the completion feature because I still use that and the completion window.

![2024-05-30_22-19](https://github.com/hrsh7th/nvim-cmp/assets/154771146/a073949b-2aaf-4c55-ab17-90a309df3e23)

**Comment by Shougo** [comment, marked as answer]
What is your config?
It seems ghost text feature is enabled.

https://github.com/hrsh7th/nvim-cmp/blob/5260e5e8ecadaf13e6b82cf867a909f54e15fd07/lua/cmp/config/default.lua#L93

https://github.com/hrsh7th/nvim-cmp/blob/5260e5e8ecadaf13e6b82cf867a909f54e15fd07/doc/cmp.txt#L182

**Reply by ghost** [reply]
Perfect setting it to false in my config worked, thanks so much!
